cpp/include/gpu_util.h

```diff
@@ -116,6 +116,33 @@ cusolverStatus_t GpuLinearSolver(cusolverDnHandle_t handle,
                                  int ldb,
                                  int *devInfo);
 
+cusolverStatus_t do_get_det_buffer(cusolverDnHandle_t handle,
```

Comment:
Could you change the function name to camel case with Gpu as prefix? For example, do_get_det_buffer should be GpuGetDeterminantBuffer or GpuGetDeterminantWorkspace. So are the other wrapper functions. Many thanks!